skbio/util/_testing.py

```diff
@@ -349,6 +349,23 @@ def assert_data_frame_almost_equal(left, right, rtol=1e-5):
     assert_index_equal(left.index, right.index)
 
 
+def _data_frame_to_default_int_type(df):
+    """Convert integer columns in a data frame into the platform-default integer type.
+
+    Pandas DataFrame defaults to int64 when reading integers, rather than respecting
+    the platform default (Linux and MacOS: int64, Windows: int32). This causes issues
+    in comparing observed and expected data frames in Windows. This function repairs
+    the issue by converting int64 columns of a data frame into int32 in Windows.
+
+    See: https://github.com/unionai-oss/pandera/issues/726
+
+    """
+    for col in df.select_dtypes("int").columns:
+        df[col] = df[col].astype(int)
+
+    return df
```

Comment:
This function doesn't have to return `df` -- it manipulates `df` in place. Returning `df` can actually confuse the user, who might assume that the function creates a copy while not touching the original input.

Reply:
Without the `return` statement, the tests fail with the following error:

`AssertionError: DataFrame Expected type <class 'pandas.core.frame.DataFrame'>, found <class 'NoneType'> instead`